### DISCUSSION TITLE
Case insensitive string filter

### DISCUSSION THREAD

**Discussion by Xyphyn** [pocketbase/pocketbase#2201] [Q&A] [answered]
I'm trying to fetch a user from the users collection based on username. Is there a way to filter strings case insensitive? Doing `username = "JAMES"` won't work if the user's name is `James`.

**Comment by ganigeorgiev** [comment, marked as answer]
One workaround could be to use the "contains/like" (`~`) operator.

Another alternative is [Dao.FindAuthRecordByUsername](https://pkg.go.dev/github.com/pocketbase/pocketbase@v0.14.1/daos#Dao.FindAuthRecordByUsername).

**Reply by Xyphyn** [reply]
Alright, thanks

**Comment by pgswow** [comment]
@ganigeorgiev Hello! (`~`) operator for filters work only Eng/Latin letters?

**Reply by ganigeorgiev** [reply]
@pgswow Yes, currently it is case insensitive only for ASCII characters as we don't use the ICU extension. You can also check https://github.com/pocketbase/pocketbase/issues/627.

If you use PocketBase as framework, you can enable the ICU extension with the CGO driver:

```sh
CGO_ENABLED=1 go build -tags "sqlite_icu"
```

**Reply by pgswow** [reply]
Thanks! =)
